templates/pull_request_template.md

```diff
@@ -0,0 +1,20 @@
+All Submissions:
+
+* [ ] Have you followed the guidelines in our Contributing document?
+* [ ] Have you checked to ensure there aren't other open [Pull Requests](../../../pulls) for the same update/change?
+
+<!-- You can erase any parts of this template not applicable to your Pull Request. -->
+### Checklist before requesting a review
+- [ ] I have performed a self-review of my code
+- [ ] If it is a core feature, I have added thorough tests.
```

Comment:
Sonar should flag if coverage is low, maybe we could take out the core feature part?
```suggestion
- [ ] I have added thorough tests, if appropriate.
```